### PR TITLE
Fill the model with internal attributes after create/saving

### LINKED ISSUE #390: Model no attribute 'id' after creating
**Describe the bug**

Storing a new model does not allow to access the `id` attribute. Accessing in code after `save()` results in a 
`class model 'Category' has no attribute id`

```python
category.save()
request.session.flash('success', f'Saved: {category.id}')
return request.redirect_to('admin.category.table')
```

**To Reproduce**

create a simple store function:

```python
from app.Category import Category

class CategoryController(Controller):

    def store(self, request: Request):
        category = Category()
        category.name = request.input('name')
        category.slug = request.input('slug')
        category.description = request.input('description')
        category.sorting = request.input('sorting')
        category.save()
        return category
```

**Expected behavior**
What do you believe should be happening?
```json
{
  "id": 1,
  "name": "test",
  "slug": "test",
  "description": "test",
  "sorting": "1"
}
```

**Actual behavior**
```json
{
  "name": "test",
  "slug": "test",
  "description": "test",
  "sorting": "1"
}
```

**Screenshots or code snippets**
**What database are you using?**
 - Type: SQLite 2.8.17
 - Masonite ORM:  masonite-orm==1.0.27

### PR DESCRIPTION
**Pull request by Marlysson**
Closes #390 
@josephmancuso As this is too simple, I submitted soon.

### CODE REVIEW

**Comment by josephmancuso** [comment]
Did that fix the issue?

**Comment by Marlysson** [comment]
yeah it does.

File: src/masoniteorm/models/Model.py [hunk 1]
```diff
@@ -590,12 +590,14 @@ def save(self, query=False):
             else:
                 result = self.create(self.__dirty_attributes__, query=query)
             self.observe_events(self, "saved")
+            self.fill(result.__attributes__)
```
Comment (josephmancuso) [inline src/masoniteorm/models/Model.py:593]: I don't think it is `__attributes__` right? Because the result of the create will contain the `id` ... 🤔

**Review by josephmancuso — CHANGES_REQUESTED** [review]

File: src/masoniteorm/models/Model.py (hunk 1 above)
Comment (Marlysson) [inline src/masoniteorm/models/Model.py:593]: Passing the whole "result" doesn't work, it receive a dictionary.

Comment (josephmancuso) [reply src/masoniteorm/models/Model.py:593]: shouldn't it be:

```python
self.fill(result)
```

from the result of the create, which contains the `id` of the create we just created

**Review by josephmancuso — COMMENTED** [review]
OOHH I see. you are doing `result.__attributes__` because it is a model that gets returned ..

File: src/masoniteorm/models/Model.py (hunk 1 above)
Comment (Marlysson) [reply src/masoniteorm/models/Model.py:593]: When I used the self.fill(result) it raises this in `fill` method:

```python
self.__attributes__.update(attributes)
...
TypeError: getattr(): attribute name must be string
```

If I'm not wrong the type of "result" is a Model ( Category) , then I use the atributes returned by result.

**Comment by Marlysson** [comment]
Exactly. There other cases?

**Comment by josephmancuso** [comment]
No not when working with models I think this works perfect. Its been a long day lol. I was reading it as `self.__attributes__` instead of `result.__attributes__`. My eyes are tired

**Review by josephmancuso — APPROVED** [review]

**Pull request merged**